README.md

```diff
@@ -54,6 +54,13 @@ python -m venv venv
 
 After the initial setup, you can proceed with installing whotracks.me.
 
+For nushell:
+
+```nushell
+python -m virtualenv venv
```

Comment:
Why virtualenv and not venv (part of the standard)?

Reply:
`venv` does not support `nushell`